fladgejt/login.py

```diff
@@ -45,7 +45,7 @@ def get_cosign_cookies(server, params, logger):
 
     if params['type'] == 'cosignpassword':
         if 'ais_url' in server:
-            url = server['ais_url'] + 'ais/login.do?'
```

Comment:
@Adman as per our discussions elsewhere, let's move this `login` vs `loginCosign` distinction to settings.

Reply:
Do you mean votrfront settings, or create separate settings for fladgejt?